app/src/main/java/fr/free/nrw/commons/review/ReviewPagerAdapter.java

```diff
@@ -46,4 +46,10 @@ public Fragment getItem(int position) {
         return reviewImageFragments[position];
     }
 
+    public void enableButtons(){
+        reviewImageFragments[0].enableButtons();
+    }
```

Comment:
null check can be beneficial here

Reply:
We use braces for single line if too, see our https://github.com/commons-app/apps-android-commons/wiki/Code-style and https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used